tailwind.config.ts

```diff
@@ -27,6 +28,34 @@ export default {
             '3xl': `${TRIPLE_XL_SCREEN_SIZE}px`
         },
         extend: {
+            aspectRatio: {
+                1: '1',
+            },
+            colors: {
+                'teal-950': '#0A3B2F',
+                'teal-900': '#114E3F',
+                'teal-800': '#1E6C59',
+                'stone-900': '#1D1D1D',
+                'stone-500': '#F8F8F8',
```

Comment:
We are not using the material design convention
- primary-color
- secondary-color
- tertiary-color
etc... see [link](https://material-foundation.github.io/material-theme-builder/)